modules/ossm-federation-config-meshPeer.adoc

```diff
@@ -0,0 +1,111 @@
+////
+This module included in the following assemblies:
+* service_mesh/v2x/ossm-federation.adoc
+////
+
+[id="ossm-task_{context}"]
+= Joining a federated mesh
+
+The federation between two meshes is declared by a `ServiceMeshPeer` resource that defines the federation from one mesh to the other and is used to configure discovery for the peer mesh, access to the peer mesh, and certificates used to validate the other mesh’s clients.
```

Comment:
To improve readability, would it help to rewrite this as multiple sentences?
Would it help to rephrased this in terms of user outcomes instead of the feature? For example: 
To <achieve goal A>, you can <summary description of task>. <elaboration>.